site/zenodo_rdm/custom_fields/community_fields.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Custom fields."""
+from invenio_i18n import lazy_gettext as _
+from invenio_vocabularies.contrib.subjects.api import Subject
+from invenio_vocabularies.contrib.subjects.schema import SubjectRelationSchema
+from invenio_vocabularies.services.custom_fields import VocabularyCF
+
+
+class SubjectCF(VocabularyCF):
```

Comment:
shouldn't this class go to invenio-vocabularies?

Reply:
As discussed IRL in the stand-up, agree, this can be moved up the chain.